x/observer/keeper/msg_server_update_gas_price_increase_flags.go

```diff
@@ -0,0 +1,52 @@
+package keeper
+
+import (
+	"context"
+
+	cosmoserrors "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authoritytypes "github.com/zeta-chain/zetacore/x/authority/types"
+	"github.com/zeta-chain/zetacore/x/observer/types"
+)
+
+func (k msgServer) UpdateGasPriceIncreaseFlags(goCtx context.Context, msg *types.MsgUpdateGasPriceIncreaseFlags) (*types.MsgUpdateGasPriceIncreaseFlagsResponse, error) {
```

Comment:
I considered that while working on this ,but Inbound and Outbound are single values where as GasPrice is a set of values , It might make sense to group the inbound and outbound flags under CCTX flags in the future but I don't think its necessary for this pr .